docs/source/topic-guides/models.md

```diff
@@ -33,6 +33,11 @@ Every create transaction contains exactly one fulfillment-condition pair. A tran
 
 ![Tracking the stories of three assets](../_static/stories_3_assets.png)
 
+To determine the current owner(s) of an asset, find the most recent valid transaction involving it, and then look at the list of owners in the _conditions_ (not the fulfillments).
```

Comment:
While that's true, it answers a different question: What assets have this public key listed as an owner (or co-owner) right now?